goaviatrix/spoke_vpc.go

```diff
@@ -513,3 +515,8 @@ func (c *Client) DisableGlobalVpc(gateway *Gateway) error {
 	}
 	return c.PostAPI(form["action"], form, BasicCheck)
 }
+
+// diff suppress function for bgp_neighbor_status_polling_time. Suppress the diff if old value is 0
+func DiffSuppressFuncBgpNeighborStatusPollingTime(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
Can we just write an inline function for this? Seems overkill to have this function.

Reply:
sure will add that change.